### ISSUE TITLE
fix: argument order bug in _generate() input validation — crashes on non-string input

### ISSUE THREAD

**Issue by kuishou68** [OpenBMB/VoxCPM#228]
### Description

In `src/voxcpm/core.py`, line 203, the input validation guard has a **logic order bug**:

```python
if not text.strip() or not isinstance(text, str):
```

The `text.strip()` call is evaluated **before** the `isinstance(text, str)` check. If `text` is not a string (e.g. an `int`, `None`, or any other type), Python will raise an `AttributeError` when calling `.strip()` — the isinstance guard never gets a chance to catch it.

### Expected behaviour

A helpful `ValueError` with the message "target text must be a non-empty string" should be raised for any non-string input.

### Actual behaviour

When `text` is not a string (e.g. `model.generate(text=42, ...)`), the call crashes with:
```
AttributeError: 'int' object has no attribute 'strip'
```

### Root Cause

The operands of the short-circuit `or` are in the wrong order. The type check must come first.

### Fix

```python
# Before (buggy)
if not text.strip() or not isinstance(text, str):

# After (correct)
if not isinstance(text, str) or not text.strip():
```

### Affected File

`src/voxcpm/core.py`, line 203 in `_generate()`

**Comment by kuishou68** [comment]
I've opened PR #229 to fix this issue. The one-line change swaps the `isinstance` and `.strip()` checks so the type guard always runs before calling any string method, preventing `AttributeError` for non-string input. https://github.com/OpenBMB/VoxCPM/pull/229
